packages/atlas/src/components/_comments/Comment/Comment.tsx

```diff
@@ -0,0 +1,142 @@
+import { format } from 'date-fns'
+import React from 'react'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+
+import { Text } from '@/components/Text'
+import { Tooltip } from '@/components/Tooltip'
+import { SvgActionEdit, SvgActionMore, SvgActionTrash } from '@/components/_icons'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { ContextMenu } from '@/components/_overlays/ContextMenu'
+import { cVar, transitions } from '@/styles'
+import { formatDate, formatDateAgo } from '@/utils/time'
+
+import {
+  CommentBody,
+  CommentContent,
+  CommentHeader,
+  CommentHeaderDot,
+  CommentWrapper,
+  HighlightableText,
+  KebabMenuIconButton,
+  StyledSvgActionTrash,
+} from './Comment.styles'
+
+import { CommentRow, CommentRowProps } from '../CommentRow'
+
+export type CommentProps = {
+  memberHandle?: string
+  createdAt?: Date
+  comment?: string
+  loading?: boolean
+  isEdited?: boolean
+  type: 'default' | 'deleted' | 'options'
```

Comment:
I'm thinking, shouldn't we use some `isOwner` flag instead of `options` type? WDYT?

Reply:
I'm not sure, I would rather stick to the types defined by designers in this case. Remember that also the creator of the video can delete a comment.